server/modules/category/service/categoryImportCSVParser.js

```diff
@@ -62,7 +62,16 @@ const createImportSummary = async (filePath) => {
 
       const level = getOrCreateLevel(columnName, columnType)
 
-      acc[columnName] = CategoryImportSummary.newColumn(columnType, level.name, level.index)
+      const extraDataType = columnType === CategoryImportSummary.columnTypes.extra
+        ? CategoryImportSummary.columnDataTypes.text
+        : null
+
+      const columnProp = columnProps[columnType]
+      const language = columnProp && columnProp.lang
```

Comment:
why shouldn't you have a columProp here?

Reply:
in the case of "extra" columnType